poly-benches/benches/fft.rs

```diff
@@ -20,12 +20,6 @@ const BENCHMARK_LOG_INTERVAL_DEGREE: usize = 1;
 const ENABLE_RADIX2_BENCHES: bool = true;
 const ENABLE_MIXED_RADIX_BENCHES: bool = true;
 
-const ENABLE_SUBGROUP_FFT_BENCH: bool = true;
```

Comment:
i thought these flags were useful, so that I could just bench the function I was working on

Reply:
You can still do that by passing an argument to cargo bench. For example, `cargo bench ifft` runs only the `ifft` benchmarks. (You can pass an arbitrary regex in place of `ifft`.)